packages/algob/src/types.ts

```diff
@@ -431,6 +431,15 @@ export interface SSCCallsParam extends SSCOptionalFlags, Sign {
   payFlags: TxParams
 }
 
+export interface SSCStateSchema {
+  key: string
+  value: {
+    type: number
+    bytes: string
+    uint: number
+  }
+}
```

Comment:
should we move it to `types-algosdk`? If yes, then please merge master before doing it to avoid conflicts.

Reply:
ok